packages/api/src/parse-cli.ts

```diff
@@ -316,6 +316,11 @@ export default function parseCli(argv?: string | readonly string[]) {
           "base stream name to be used in wildcard-based routing scheme.",
         type: "string",
       },
+      "recording-source-url": {
+        describe:
+          "base stream name to be used in wildcard-based routing scheme.",
```

Comment:
The description is outdated

Reply:
Updated in be2a1f64d7ccf309584a69035295d88989833208